test/api-test.js

```diff
@@ -435,13 +435,19 @@ describe('RippleAPI', function () {
           "maxLedgerVersion": 8820051
         }
       }
-
+      
       return this.api.preparePayment(
         address, requests.preparePayment.normal, localInstructions).then(
           _.partial(checkResult, expectedResponse, 'prepare'));
     });
   });
 
+  it('createPaymentTransaction - normal', function(){
+    return this.api.createPaymentTransaction(address, requests.preparePayment.normal).then((res)=>{
+      assert.deepStrictEqual(res.txJSON,  "{\"TransactionType\":\"Payment\",\"Account\":\"r9cZA1mLK5R5Am25ArfXFmqgNwjZgnfk59\",\"Destination\":\"rpZc4mVfWUif9CRoHRKKcmhu1nx2xktxBo\",\"Amount\":{\"currency\":\"USD\",\"issuer\":\"rMH4UxPrbuMa1spCBR98hLLyNJp4d8p4tM\",\"value\":\"0.01\"},\"Flags\":2147483648,\"SendMax\":{\"currency\":\"USD\",\"issuer\":\"rMH4UxPrbuMa1spCBR98hLLyNJp4d8p4tM\",\"value\":\"0.01\"},\"LastLedgerSequence\":8819954,\"Sequence\":23,\"Fee\":\"12\"}")
```

Comment:
`deepStrictEqual` doesn't exist. Also, `createPaymentTransaction` returns a txJSON object, not an object containing a string called `txJSON`. So you'll need to do something like:
```
  it('createPaymentTransaction - normal', function(){
    const res = this.api.createPaymentTransaction(address, requests.preparePayment.normal)
    assert.deepEqual(res, JSON.parse("{\"TransactionType\":\"Payment\",\"Account\":\"r9cZA1mLK5R5Am25ArfXFmqgNwjZgnfk59\",\"Destination\":\"rpZc4mVfWUif9CRoHRKKcmhu1nx2xktxBo\",\"Amount\":{\"currency\":\"USD\",\"issuer\":\"rMH4UxPrbuMa1spCBR98hLLyNJp4d8p4tM\",\"value\":\"0.01\"},\"Flags\":2147483648,\"SendMax\":{\"currency\":\"USD\",\"issuer\":\"rMH4UxPrbuMa1spCBR98hLLyNJp4d8p4tM\",\"value\":\"0.01\"},\"LastLedgerSequence\":8819954,\"Sequence\":23,\"Fee\":\"12\"}"))
  })
```
Notice that `createPaymentTransaction` doesn't set `LastLedgerSequence`, `Sequence`, or `Fee`.